docs/community-resources/developer-portal/index.md

```diff
@@ -0,0 +1,270 @@
+</Callout>
+
+### Using DocCardList
+
+Using `DocCardList` improves the layout of the index page:
+
+<Callout type="note" title="Example">
+![edit page](./images/doc-card-list-light.png#gh-light-mode-only)
+![edit page](./images/doc-card-list-dark.png#gh-dark-mode-only)
+</Callout>
+
+On the most pages you can use just `<DocCardList />` component imported from `'@theme/DocCardList'`
+
+```markdown
+import DocCardList from '@theme/DocCardList';
+
+# Deployments
+
+This section contains guides to deploying and testing smart contracts.
+
+## All Sections
+<DocCardList />
+
+```
+
+On the top level index category pages you have to use `useDocsSidebar` react hook to avoid rendering errors
+
+```markdown
+import DocCardList from '@theme/DocCardList';
+import { useDocsSidebar, isSamePath } from '@docusaurus/theme-common/internal';
+import { useLocation } from '@docusaurus/router';
+
+<DocCardList items={useDocsSidebar().items.filter(item => !isSamePath(item.href, useLocation().pathname))}/>
+```
+
+<Callout type="warning">
+If you use `<DocCardList/>` on the top level category index page (e.g. `./docs/tools/index.mdx`), Docusaurus will throw an error:
+
+> useCurrentSidebarCategory() should only be used on category index pages.
+
+</Callout>
+
+## SEO
+
+Basic SEO metadata can be included in markdown `frontmatter`. The developer Portal supports `title` and `description` metadata. If none is provided, a default will be applied.
+
+Example `frontmatter`:
+
+```markdown
+---
+title: "Hello World"
+description: "A Great Document"
+---
+```
+
+It is not required to supply this SEO metadata. Flow's in-house SEO experts will provide custom search-engine appropriate metadata for each page. Your custom metadata will be used in lieu of metadata supplied by our SEO experts.
+
+
+## Page Content
+
+
+### Images and Other Media
+
+Include images and media using relative URLs within the `docs` folder. If your media is viewable in GitHub, then it should display on the Developer Portal.
+
+For external media or URLs, use fully qualified URLs, eg:
+
+`![image](https://github.com/onflow/cadence/images/image.png)`
+
+The same rule applies to all other external media.
+
+### Links
+
+Linking to documents within the `docs` folder of your repo can be done in the standard way that is acceptable to GitHub, eg.
+
+```
+[link](./doc-two.md#hash-link)
+```
+
+Writing links in your documents is easy if you follow this rule-of-thumb: If it works in GitHub it should work on the Developer Portal, with one notable exception.
+
+**Links to content outside the `docs` folder** in your repository, or elsewhere on the web must be in the form of a fully-qualified URL, eg:
+
+```
+[link](https://www.google.com)
+```
+
+<Callout type="tip">
+Use relative links directly to .md/.mdx files
+</Callout>
+
+### Callouts
+
+Many pages have callouts/admonitions. It is a slightly customized version of [standard admonitions](https://docusaurus.io/docs/markdown-features/admonitions)
```

Comment:
for quick reference could you mention some of the Callout types like the ones already used in this doc. 
warning, tip, info, success